galaxy/tools/deps/mulled/mulled_build.py

```diff
@@ -285,7 +286,9 @@ def exec_command(self, involucro_args):
         return res
 
     def is_installed(self):
-        return os.path.exists(self.involucro_bin)
+        if find_executable('involucro') is not None or os.path.exists(self.involucro_bin):
```

Comment:
`distutils.spawn.find_executable` only does half the job of `shutil.which`, i.e., it is only checking for existence but not user executability.
I think (given a `shutil.which` implementation) you'd want something like
```python
if not os.path.dirname(self.involucro_bin):
    return shutil.which(self.involucro_bin)
return (os.access(self.involucro_bin, os.F_OK | os.X_OK) and not os.path.isdir(self.involucro_bin))
```
```